cmd/cluster-init/update_jobs.go

```diff
@@ -160,16 +164,71 @@ func generateSecretVolume(clusterName string) v1.Volume {
 	}
 }
 
-func generateContainer(image, clusterName string, extraArgs []string, extraVolumeMounts []v1.VolumeMount) v1.Container {
+func generateContainer(image, clusterName string, extraArgs []string, extraVolumeMounts []v1.VolumeMount, home bool) v1.Container {
```

Comment:
rather than the `home` arg can we do a `extraEnvVars []v1.EnvVar` similar to the rest of the signature?

Reply:
Updated.